src/plots/polar/index.js

```diff
@@ -66,6 +69,11 @@ function clean(newFullData, newFullLayout, oldFullData, oldFullLayout) {
                 oldSubplot.clipPaths[k].remove();
             }
         }
+
+        if(mustCleanScene && oldSubplot._scene) {
+            oldSubplot._scene.destroy();
+            oldSubplot._scene = null;
+        }
```

Comment:
Does it work correctly if there's a separate `scattergl` or `scatter3d` subplot when you flip between `scatterpolar` and `scatterpolargl`?

Reply:
Yes it does work correctly :lock: in -> https://github.com/plotly/plotly.js/pull/3098/commits/cab319d48b4d602223127842f9ccea84273b62bc with some explanations.